app/controllers/translation_controller.rb

```diff
@@ -123,8 +123,8 @@ def update_translations(tags)
       elsif old_val != new_val
         change_translation(str, new_val)
         any_changes = true
-      elsif
-        touch_translation(str) # rubocop:disable Layout/ConditionPosition
+      else
+        touch_translation(str)
```

Comment:
**Does this work?**
Fixes Lint/EmptyConditionalBody offense.

Reply:
Yes, clearly a typo, thanks!